component/src/main/java/me/androidbox/component/event/VisitorInfo.kt

```diff
@@ -0,0 +1,6 @@
+package me.androidbox.component.event
+
+data class VisitorInfo(
+    val visitorName: String = "",
+    val visitorId: String = ""
+)
```

Comment:
Good, just that I wouldn't use default parameters for non-nullable strings this way. If you want to indicate that a value doesn't exist, make it nullable - that's in the end what nullables are used for. In this case I would keep the strings as non-null and not initialize them, but rather work with a `VisitorInfo?`. Either a visitor exists or not, but if they exist they certainly have a name and an ID

Reply:
@philipplackner Updated the VisitorInfo and removed default arguments.